public/director-upload-movie.php

```diff
@@ -1,15 +1,76 @@
+          require("../utils/filter.php");
+          DEFINE('DS', DIRECTORY_SEPARATOR); 
+          
+          $userId = 2;
+
+          if(isset($_POST['upload'])){
+            if(!empty($_POST['MovieTitle']) AND !empty($_POST['Category']) AND !empty($_POST['AgeRating']) AND !empty($_POST['filmGuide'])  AND  !($_FILES['Movie']['error'] > 0) AND !($_FILES['Thumbnail']['error'] > 0) AND !empty($_POST['Description'])){  
+              $title = $_POST['MovieTitle'];
+              $genre = $_POST['Category'];
+              $ageRating = $_POST['AgeRating'];
+              $filmGuide = $_POST['filmGuide'];
+              $description = $_POST['Description'];
+              $moviefile = $_FILES['Movie'];
+              $thumbnailfile = $_FILES['Thumbnail'];
+              $allowed = array("mp4","mp3");
+              $moviename = $_FILES['Movie']['name'];
+              $ext = pathinfo($moviename, PATHINFO_EXTENSION);
+              if(!in_array($ext, $allowed)){
+                  echo "filetype not allowed, must be .mp4";
+              }else{
+                if(strlen($_FILES['Movie']['name']) < 70){
+                  $upload2_dir = __DIR__.DS."uploads".DS."user_".$userId;
+                  $tmpFileName = $_FILES['Movie']['tmp_name'];
+                  moveUploadedFile($upload2_dir, $tmpFileName, $moviename);
+                  $path = "$upload2_dir".DS."$moviename";
+                  $movie = TRUE;
+                }else{
+                  $movie = FALSE;
+                }
+              }
+              $allowed = array("jpeg", "jpg","png","gif");
+              $thumbnailname = $_FILES['Thumbnail']['name'];
+              $ext = pathinfo($thumbnailname, PATHINFO_EXTENSION);
+              if(!in_array($ext, $allowed)){
+                  echo "filetype not allowed, must be .png, ,jpeg or .jpg";
+              }else{
+                if(strlen($_FILES['Thumbnail']['name']) < 70){
+                  $upload_dir = __DIR__.DS."uploads".DS."user_".$userId;
+                  $tmpFileName = $_FILES['Thumbnail']['tmp_name'];
+                  moveUploadedFile($upload_dir, $tmpFileName, $thumbnailname);
+                  $thumbnailPath = "$upload_dir".DS."$thumbnailname";
+                  $thumbnail = TRUE;
+                }else{
+                  $thumbnail = FALSE;
+                }
+              }
+              if ($movie == TRUE AND $thumbnail == TRUE){
+                uploadMovie($userId, $title, $path, $thumbnailPath, $description, $ageRating, $filmGuide, $genre);  
+              }
+            }else{
+              $titlemess = "Please add a title";
+              $genremess = "Please select genres.";
+              $ageRatemess = "Please select an age rating.";
+              $filmGuidemess = "Please select film guides.";
+              $descriptionmess = "Please add a description.";
+              $moviemess = "Please add a movie file.";
+              $thumbnailmess = "Please add a thumbnail.";
```

Comment:
Op dit moment krijg ik voor alle velden een error message op het moment dat ik 0 velden, 1, 2, 3 of 5 velden ingevuld heb. Dit zou per keer 1 error message moeten terugkeren wanneer een waarde niet juist is, of de juiste error message bij het juiste veld. Dus wanneer er een veld is ingevuld, dat er dan geen foutmelding bij komt

Reply:
Wanneer ik alles invul, en bij een thumbnail in plaats van een afbeelding een video zou uploaden, krijg ik de volgende error 

![image](https://user-images.githubusercontent.com/89914058/146895094-a35889fb-b771-42e4-8c4f-8bc62a48c9b1.png)

De check is dus nog niet helemaal goed
Hetzelfde geld wanneer ik een image wil uploaden bij een movie, dan krijg ik de volgende foutmelding

![image](https://user-images.githubusercontent.com/89914058/146895365-278116d6-afef-4d3c-85aa-22d3287db6ed.png)

de eerste lijn van de error is goed, maar de 2e, waarin benoemd wordt 

"Warning: Undefined variable $movie in C:\xampp\htdocs\and-action\public\director-upload-movie.php on line 60" zou niet naar de gebruiker terug mogen keren

De plek van de error message is links bovenin dan niet goed